src/routes/_index/+layout.style.scss

```diff
@@ -21,6 +21,15 @@
 
 /* stylelint-disable no-descending-specificity */
 
+/*  
+Some browsers, especially within mobile devices, include an automatic mechanism for inflating text to help readability. We're turning that off here to get to a consistent baseline.
+*/
+html {
+    -webkit-text-size-adjust: none;
+    -moz-text-size-adjust: none;
+    text-size-adjust: none;
+}
+
```

Comment:
This seems like a great change, but not part of this PR. 
Feel free to make a separate commit and just push it up directly to 19.2.0
And no need to add a changeset for that